tests/integration/reconcile/connectors/test_read_schema.py

```diff
@@ -80,32 +80,41 @@ def _get_snowflake_options(self):
         return opts
 
 
-@pytest.mark.skip(reason="Run in acceptance environment only")
-def test_sql_server_read_schema_happy(mock_spark):
+def test_sql_server_read_schema_happy(mock_spark: SparkSession) -> None:
     mock_ws = create_autospec(WorkspaceClient)
     connector = TSQLServerDataSourceUnderTest(mock_spark, mock_ws)
 
     columns = connector.get_schema("labs_azure_sandbox_remorph", "dbo", "reconcile_in")
     assert columns
 
 
-def test_databricks_read_schema_happy(mock_spark):
+def test_databricks_read_schema_happy(mock_spark: SparkSession) -> None:
     mock_ws = create_autospec(WorkspaceClient)
     connector = DatabricksDataSource(get_dialect("databricks"), mock_spark, mock_ws, "my_secret")
 
     mock_spark.sql("CREATE DATABASE IF NOT EXISTS my_test_db")
     mock_spark.sql("CREATE TABLE IF NOT EXISTS my_test_db.my_test_table (id INT, name STRING) USING parquet")
     df = mock_spark.sql("SELECT * FROM my_test_db.my_test_table")
-    df.createGlobalTempView("my_global_test_view")
-    columns = connector.get_schema(None, "global_temp", "my_global_test_view")
+    random_view = f"test_view_{uuid.uuid4().hex}"
```

Comment:
This has triggered me. :) I see that it's trying to do, but there's more going on here. Those `IF NOT EXISTS` clauses above hint at the problem: this test doesn't seem to clean up after itself.

The pattern should be:

 - Use a randomised database/schema name.
 - Everything else with a fixed name is fine, because it's scoped.
 - Delete the database when done.

(It's unclear to me if the spark session is connected to a real workspace: if it is it should use the `make_schema` fixture.)